### PR TITLE
Feature 2007 git revision in footer

### PR DESCRIPTION
**Pull request by Algentile**
This Issue is referenced in [#2007](https://github.com/iodide-project/iodide/issues/2007) and it adds the most recent commit hash from local master branch to the footer of public iodide docs.

### Pull Request checklist

- [N / A] **Documentation**: If this feature has or requires documentation, the relevant docs have been updated.
- [ ] **Changelog**: This PR updates the [changelog](../CHANGELOG.md) with any user-visible changes.
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
--> Testing Done in production build --> IODIDE_PUBLIC = True

### CODE REVIEW

**Comment by Algentile** [comment]
Hey @wlach, I added the plugin and used the most recent commit hash from local master running under the assumption that the user would keep their branch up to date with their local master. 

I also have a slightly different variant where I check I find the most recent commit hash that is the intersection between master and feature branch in the case in the case that the user might not update their local feature branch right away.

**Comment by wlach** [comment]
Awesome, thank you! I've run out of time to look at this today but will check it out next week.

File: src/shared/components/footer.jsx [hunk 1]
```diff
@@ -43,10 +43,18 @@ const Footer = ({ showIcon = true }) => (
           .
         </p>
       )}
-      {// only display terms of service on an official mozilla installation
-      process.env.IODIDE_PUBLIC && (
+      {process.env.IODIDE_PUBLIC && (
+        // only display terms of service on an official mozilla installation
         <ul>
           <li>Alpha Software</li>
+          <li>
```
Comment (wlach) [inline src/shared/components/footer.jsx]: We should incorporate the revision into the same li tag as "Alpha Software", adding a seperate tag adds unnecessary whitespace.

**Comment by wlach** [comment]
One other minor note, could you give a descriptive title for the PR title. If you want to reference an issue, put it in parantheses and give it a hash like this `(fixes #2007)`. A good title for this PR would be "Add git revision to footer on iodide pages"

**Comment by wlach** [comment]
In any case, I've merged your changes. Thanks again for this work!

**Comment by Algentile** [comment]
Thanks @wlach for reviewing, I will make sure I make the PR titles more descriptive in the future.
